README.md

```diff
@@ -1,7 +1,11 @@
 HTTP Factories
 ==============
 
-This is the implementation of [PSR-17 (HTTP Message Factories)][psr-17]. Please refer to the
-specification for a description.
+This repository holds all interfaces related to [PSR-17 (HTTP Message Factories)][psr-17]. 
+Please refer to the specification for a description.
+
+You can find implementations of the specification by looking for packages providing the 
+[psr/http-factory-implementation](https://packagist.org/providers/psr/http-factory-implementation) virtual package.
```

Comment:
Maybe we can point the link directly to the dependents of that package?

Reply:
Why to the dependents? From my perspective, visitor of this repository will mostly be interested in implementations.

Reply:
Uh sorry, I misread the link! That is the correct one!